src/main/kotlin/blackjack/domain/DealerResult.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.domain
+
+import blackjack.entity.Dealer
+
+data class DealerResult(
+    val dealerName: String,
+    val winCount: Int,
+    val loseCount: Int,
+    val drawCount: Int,
```

Comment:
아래와 같이 작성하면 팩토리 함수로만 객체를 초기화하도록 강제해볼 수 있겠네요!

```suggestion
data class DealerResult private constructor(
    val dealerName: String,
    val winCount: Int,
    val loseCount: Int,
    val drawCount: Int,
```